MASTER FUNCTIONS LIBRARY.vbs

```diff
@@ -1540,9 +1540,12 @@ function access_ADDR_panel(access_type, notes_on_address, resi_line_one, resi_li
         mail_line_two = replace(mail_line_two, "_", "")
 		mail_street_full = trim(mail_line_one & " " & mail_line_two)
         mail_city = replace(mail_city_line, "_", "")
-        mail_state = replace(mail_state_line, "_", "")
-        mail_zip = replace(mail_zip_line, "_", "")
-
+		Call get_state_name_from_state_code(mail_state_line, mail_state, TRUE)		'This function makes the state code to be the state name written out - including the code
+		mail_state = replace(mail_state_line, "_", "")
```

Comment:
This will replace the assignment to this variable that we just had on the previous line.

Reply:
Whoops- good catch. I meant that to be mail_state for both var in that line but based on your comment below I'll plan to remove the line anyhow!